zerver/models.py

```diff
@@ -1014,6 +1014,12 @@ def can_subscribe_other_users(self) -> bool:
             return True
         return False
 
+    def can_only_administrators_subscribe_other_users(self) -> bool:
+        """
+        This function exists primarily to make testing earlier.
+        """
+        return self.realm.invite_to_stream_by_admins_only
```

Comment:
I'm not sure this helper function is worth it; we generally try to avoid 1-line helper functions that just return a value because they require folks reading the code to learn two names for the same thing.

Reply:
Fixed, thanks!